<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>811</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans Mono</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>NFA to DFA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="imgView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="predictedText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>841</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7, 119, 197);
font: 75 16pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Text_9">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Created by:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Anthony Amgad Fayek Selim : 19P9880&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Mahmoud Mohamed Elnashar : 19P3374&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7, 119, 197);
font: 75 16pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>Hover over me!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="cnnradioButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>70</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7, 119, 197);
font: 75 16pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>CNN</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="gruradioButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>100</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7, 119, 197);
font: 75 16pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>GRU</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="lstmradioButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>130</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7, 119, 197);
font: 75 16pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>LSTM</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>1101</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
